esp/esp/program/controllers/lottery.py

```diff
@@ -290,6 +290,11 @@ def fill_section(self, si, priority=False, rank=10):
         
         #   Compute number of spaces - exit if section or program is already full.  Otherwise, set num_spaces to the number of students we can add without overfilling the section or program.
         num_spaces = self.section_capacities[si] - numpy.sum(self.student_sections[:, si])
+        # TODO(benkraft): This doesn't support program_size_by_grade.  For
+        # Stanford's use case, they don't need it -- they want the cap to
+        # apply to entering the lottery too.  But we should either decide that
+        # that's the Right Way and remove the program size logic from here
+        # entirely, or implement it here too.
```

Comment:
It's not useless in the case where you have no program cap while students are entering the lottery and then set a program cap right before running the lottery script, which is what we experimented with (although didn't end up actually using, so this is maybe not the strongest argument) for Summer HSSP.